addons/ai/functions/fnc_mortarFire.sqf

```diff
@@ -0,0 +1,24 @@
+/*
+    Author:
+        Adam Bridgford
+
+    Description:
+        Fires the mortar based on POS
+
+    Parameter(s):
+        0: Unit <OBJECT>
+
+    Return Value:
+        Nothing
+*/
+
+#include "script_component.hpp"
+
+params ["_unit"];
+
+private _enemy = _unit findNearestEnemy _unit;
+if (!mortCooldown) then {
```

Comment:
Maybe look into making this a variable stored on the mortar unit? Similar to https://github.com/uksf/Modpack/pull/185/files#diff-bb83656e9820ca16087094f4d1b5b9bdR23